base/shell.jl

```diff
@@ -81,7 +81,7 @@ function shell_parse(raw::AbstractString, interp::Bool)
                     if done(s,k)
                         error("unterminated double quote")
                     end
-                    if s[k] == '"' || s[k] == '$'
+                    if s[k] == '"' || s[k] == '$' || s[k] == '\\'
```

Comment:
The example is just this string, `"\\"`. I added test cases to `spawn.jl` as part of the patch. @nolta's port already has this change, if I read the Lisp correctly.

If you want, I can split this patch into two...